### PR TITLE
Add icons for external docs in dropdown

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Created a new PR to test the proposed feature of #118 and new designs for the icons

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ const katex = require('rehype-katex');
 module.exports = {
   title: 'IOTA Wiki',
   tagline: 'The complete reference for IOTA',
-  url: 'https://wiki.iota.org',
-  baseUrl: '/',
+  url: 'https://dr-electron.github.io/',
+  baseUrl: '/iota-wiki/',
```
Comment (Dr-Electron) [inline docusaurus.config.js:10]: Needs to be reverted

File: .github/workflows/gh-pages.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 name: deploy
 
 on:
-  pull_request:
-    branches: [develop]
+  push:
+    branches: [feat/add-dropdown-icons]
```
Comment (Dr-Electron) [inline .github/workflows/gh-pages.yml:5]: Needs to be reverted

File: src/theme/NavbarItem/DefaultNavbarItem.js [hunk 3]
```diff
@@ -30,6 +33,30 @@ export function NavLink({
...
   const isExternalLink = label && href && !isInternalUrl(href);
   const isDropdownLink = activeClassName === dropdownLinkActiveClass;
+  const { isDarkTheme } = useThemeContext();
+
+  let labelContainerStyle = null;
+
+    /**
+     * If we have a labelIcon, create the element and modify the container styles.
+     */
+  let labelIcon;
```
Comment (Dr-Electron) [inline src/theme/NavbarItem/DefaultNavbarItem.js:43]: Maybe I should set this to null to be consistent? @lucas-tortora ?

**Review by Dr-Electron — COMMENTED** [review]
Not sure if we als already want to implement the theming ina better way...
Otherwise looks fine to me after the marked things are reverted

**Review by jlvandenhout — APPROVED** [review]
Just nitpicking a little, but I would either write out "Library" in full instead of "Lib" or leave it out entirely as they are part of the Libraries category. But that just something to consider. Other than that: awesome!

**Pull request merged**